tests/test_feedexport.py

```diff
@@ -29,6 +29,8 @@
 from scrapy.utils.python import to_native_str
 from scrapy.utils.project import get_project_settings
 
+from pathlib2 import Path
```

Comment:
You should use `pathlib.Path` from the standard library, not from this third-party package.